scripts/ci/kubernetes/ci_run_helm_testing.sh

```diff
@@ -18,7 +18,28 @@
 
 echo "Running helm tests"
 
+. "$( dirname "${BASH_SOURCE[0]}" )/../libraries/_script_init.sh"
+
+if [[ -f "${BUILD_CACHE_DIR}/.skip_tests" ]]; then
+    echo
+    echo "Skip tests"
+    echo
+    exit
+fi
+
+build_images::prepare_ci_build
+
+build_images::rebuild_ci_image_if_needed
+kind::make_sure_kubernetes_tools_are_installed
+
+
+python -m pip install pytest kubernetes \
```

Comment:
Can you look here? https://github.com/apache/airflow/blob/master/scripts/in_container/entrypoint_ci.sh#L232-L277
It seems to me that you should generalize the test run environment.

Reply:
@mik-laj Is this necessary? I'm pretty heavily basing this on how we currently do kubernetes testing https://github.com/apache/airflow/blob/master/scripts/ci/kubernetes/ci_run_kubernetes_tests.sh This seems like a pretty one-off situation in a similar fashion (doesn't really require anything from the airflow CI and in reality besides the k8s library the testing environment should be very basic).

Reply:
I am concerned that we will be able to achieve a selective selection of tests based on directories, if we do not have a unified setup. Now we will only be able to run the Chart tests if there are changes to Chart only. This will greatly reduce the frequency of running these tests.

The ability to run in Breeze will also facilitate contributions by new contributors. They will be able to run the tests very easily via ``breeze tests -- /chart/tests/``. 

While I'm not sure about the first part, we should definitely support running the tests in Breeze.  This shouldn't be difficult as we just need Helm to have Docker installed.

Reply:
When I did following changes:
```diff
Author: Kamil Breguła <mik-laj@users.noreply.github.com>
Date:   Sat Oct 24 03:00:24 2020 +0200

    Add Helm to dockerfile

diff --git a/Dockerfile.ci b/Dockerfile.ci
index e352bedc3..e2528f748 100644
--- a/Dockerfile.ci
+++ b/Dockerfile.ci
@@ -215,6 +215,14 @@ RUN mkdir -p /opt/bats/lib/bats-file \

 RUN echo "export PATH=/opt/bats/bin:${PATH}" >> /root/.bashrc

+# Install HElm
+ARG HELM_VERSION="v3.2.4"
+
+RUN SYSTEM=$(uname -s | tr '[:upper:]' '[:lower:]') \
+    && HELM_URL="https://get.helm.sh/helm-${HELM_VERSION}-${SYSTEM}-amd64.tar.gz" \
+    && curl --location ${HELM_URL} | tar -xvz -O ${SYSTEM}-amd64/helm > /usr/local/bin/helm \
+    && chmod +x /usr/local/bin/helm
+
 # Additional scripts for managing BATS addons
 COPY scripts/docker/load.bash /opt/bats/lib/
 RUN chmod a+x /opt/bats/lib/load.bash
```

I was able to run the tests with one command:
```
./breeze tests  -- chart/tests
```
This is my perfect workflow.  I don't need to know any additional commands to run these tests.

Reply:
<img width="1440" alt="Screenshot 2020-10-24 at 03 04 02" src="https://user-images.githubusercontent.com/12058428/97064367-9c3d9200-15a5-11eb-82d1-797f447bf926.png">

Reply:
@mik-laj great suggestion. I have integrated it into the breeze environment so now users can run that command (and the CI will use a similar path to all other python docker-based testing).